docs/user/get-started.md

```diff
@@ -0,0 +1,339 @@
+# Get Started
+
+For the one who is interested in Vald, this article will show you how to deploy and run Vald on your k8s cluster.
+This article uses Scylla DB as the backend data store for metadata-management and backup-manager.
+Fashion-mnist is used as an example of a dataset.
+
+1. [About](#About)
+    1. [Main Features](#Main-Features)
+    2. [Requirements](#Requirements)
+2. [Starting Vald on k8s cluster](#Starting-Vald-on-k8s-cluster)
+    1. [Deploy](#Deploy)
+    2. [Run](#Run)
+3. [Advanced](#Advanced)
+
+## About
+
+Vald is a distributed highly scalable and fast approximate nearest neighbor dense feature vector search engine.<br>
+Vald can handle large amount of specific objects, such as image, text, video, and etc.<br>
+Vald is designed base on Cloud Native.
+It uses the fastest ANN Algorithm [NGT](https://github.com/yahoojapan/NGT) to search neighbors.
+(If you are interested in ANN benchmarks, please refer to [the official website](http://ann-benchmarks.com/).)
+
+### Main Features
+
+- Auto Indexing
+    - Normally, when changing the Graph Index, the Graph must be locked, but Vald uses distributed index graph, it is extremely difficult for the user to reconstruct the Graph.
+    - Therefore, Vald automatically indexes distributed Graphs sequentially.
+
+- Ingress/Egress Filltering
+    - Vald has it's own highly customizable Ingress/Egress filter.
+    - Which can be configured by the user to fit the gRPC interface.
+        - Ingress Filter: Ability to Vectorize through filter on request.
+        - Egress Filter: A function to rerank or filter the Search response with your own algorithm.
+
+- Horizontal Scaling
+    - Vald is a cloud-native vector search engine running on Kubernetes, which enables horizontal scalling of memory and cpu for billion scale of vector data.
+
+- Auto Index Backup
+    - Vald has auto index backup feature using MySQL + Redis or Cassndora which enables disaster recovery.
+
+- Distributed Indexing
+    - Vald distribute indexes vecotr to multiple agent. Which means each agent has different graph index.
+
+- Index Replication
+    - Vald stores each index in multiple agents which enables index replicas.
+    - Rebalance the replica when some pods go down.
+
+- Easy to use
+    - Vald can be easily installed in a few steps and Vald is highly customizable.
+
+### Requirements
+
+- k8s:  v1.17 ~
+- go:   v1.14 ~
+- helm: v3 ~
+- libhdf5 (_`libhdf5` is only required doing Get-Started, is not required Vald._)
+
+If helm/hdf5 is not installed, please install helm (see below details or [here](https://helm.sh/docs/intro/install))/hdf5 (see below details or [here](https://www.hdfgroup.org/)).
+
+<details><summary>optional installation</summary><br>
```

Comment:
```suggestion
<details><summary>Install Helm</summary><br>
```

Reply:
This section is optional how can we describe about it?

Reply:
How about this?
```suggestion
<details><summary>[Optional] Install Helm</summary><br>```